tests/regressions.rs

```diff
@@ -61,6 +61,7 @@ fn issue_776() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Does this fail?

Reply:
Ah, I need to remove this in a separate PR probably... the test is not correct. Spurious wakeups should be permitted.